modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -398,7 +399,19 @@ def insert(df, **kwargs):
             df["new_float64"] = np.float64(10.1)
             return df
 
-        run_and_compare(insert, data=self.data)
+        run_and_compare(applier, data=self.data)
+
+    def test_insert(self):
+        def applier(df, **kwargs):
+            df.insert(0, "new_int", 10)
+            df.insert(0, "new_float", 5.5)
+            df.insert(0, "new_list_like", np.arange(len(df)))
+            df.insert(0, "qc_column", df["new_int"])
+            return df
+
+        # setting `force_laze=False`, because `to_pandas` is not supported
+        # in lazy frame mode
+        run_and_compare(applier, data=self.data, force_lazy=False)
```

Comment:
@ienkovich Am I right that `default_to_pandas` is not supported in lazy execution mode?
https://github.com/modin-project/modin/blob/f46382c31d2bea7706e6242143817f955e1eac52/modin/experimental/engines/omnisci_on_ray/frame/data.py#L1090-L1095

Reply:
Right, we set 'lazy' mode in tests to make sure computations are done lazily and `default_to_pandas` calls are not lazy.